src/radon/types.js

```diff
@@ -25,7 +25,7 @@ const REDUCERS = {
   mode: 0x02,
   averageMean: 0x03,
   //averageMeanWeighted: 0x04,
-  //averageMedian: 0x05,
```

Comment:
For RNG requests we also need the reducer `hashConcatenate: 0x0B`

Reply:
Done!